server/src/main/java/org/cloudfoundry/identity/uaa/resources/JoinAttributeNameMapper.java

```diff
@@ -0,0 +1,28 @@
+package org.cloudfoundry.identity.uaa.resources;
+
+public class JoinAttributeNameMapper implements AttributeNameMapper {
```

Comment:
From the name of this, I can't tell what the purpose of this class is. Maybe explain in a comment?

Reply:
Added comment